### ISSUE TITLE
Support for node-inspector

### ISSUE THREAD

**Issue by simonexmachina** [trungfinity/traceur-source-maps#6]
First of all, thank you SO MUCH for this plugin. The wrong line numbers has always annoyed me about Traceur, so it's great to have a solution.

When I use this with [node-inspector](https://github.com/node-inspector/node-inspector) it tries to load the source code and generates the following errors in the console:

```
Network.loadResourceForFrontend failed.
Error: ENOENT, open '/Users/simonwade/dev/Workspace/franq/modules/basement/<compileOutput>'
```

Is this something you're interested in fixing? Is there any way I can help to sort this out?

**Comment by trungfinity** [comment]
I think this issue was fixed in commit #4. What version are you using, is it the latest version, 1.0.6?

If the version is latest and this issue still occurs, please help me in setup a small example to reproduce the issue so I can fix it.

Thank you.

**Comment by simonexmachina** [comment]
Yes, I'm using the latest version. Here's an example to reproduce:

https://github.com/aexmachina/traceur-source-maps-example

**Comment by trungfinity** [comment]
Thanks, I'm looking into it.

**Comment by tomteman** [comment]
It works with Traceur 0.0.61, but not with the latest version.

**Comment by rafaelkaufmann** [comment]
:+1: Same issue here.

**Comment by trungfinity** [comment]
Sorry for late response.

As this project is not maintained anymore, could you take a look at http://babeljs.io? I use it for daily work too and imho it's the best ES6/7 transpiler out there.
